### PR TITLE
shameless janitor buffs

### PR DESCRIPTION
**Pull request by lolman360**
putting this up as a draft for feedback

## About The Pull Request

adds a few upgraded janitor tools like /tg/ and etc have

advanced mop- craftable by the guild, takes a medium cell and is faster at mopping/sweeping + also generates its own slip-free mix with cell charge (not sure why you'd use this when the church now has the same thing but it doesn't need a cell and is 10x faster than the regular mop AND it also creates biomass)

big trashbag - craftable by anyone, it's just a 25% larger trashbag
industrial chemical sprayer - printable from the janitor design disk, 150 capacity 5 range version of the chemical sprayer that's obtainable without soteria help

todo: janitor design disk - can make mops, industrial sprayers, trashbags, lightbulbs, buckets and spray bottles

	

## Changelog
:cl:
add: shameless janitor buffs in the form of upgraded gear
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagent_containers/spray.dm [hunk 1]
```diff
@@ -179,9 +180,19 @@
...
 	return
 
+/obj/item/weapon/reagent_containers/spray/chemsprayer/industrial
+	name = "industrial chemical sprayer"
+	desc = "A utility used to spray large amounts of reagent in a given area. This is a heavy-duty industrial version, adapted to spraying chemicals such as cleaning compounds and fertilizers."
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "chemsprayerI"
+	item_state = "chemsprayerI"
+	matter = list(MATERIAL_STEEL = 15, MATERIAL_GLASS = 4, MATERIAL_PLASTIC = 8)
+	volume = 150
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/reagent_containers/spray.dm:193]:
```suggestion
	volume = 250
```

File: code/game/objects/items/weapons/mop.dm [hunk 2]
```diff
@@ -134,6 +133,50 @@
...
+	icon_state = "mop"
+	force = WEAPON_FORCE_WEAK
+	throwforce = WEAPON_FORCE_WEAK
+	throw_speed = 5
+	throw_range = 10
+	w_class = ITEM_SIZE_NORMAL
+	attack_verb = list("mopped", "bashed", "bludgeoned", "whacked")
+	matter = list(MATERIAL_PLASTIC = 12, MATERIAL_GLASS = 4, MATERIAL_STEEL = 4)
+	mopspeed  = 10
+	sweep_time = 4
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/mop.dm:149]:
```suggestion
	sweep_time = 2
```

**Review by Trilbyspaceclone — CHANGES_REQUESTED** [review]
Your still deleting icons and sprites for other things

**Comment by Trilbyspaceclone** [comment]
Your deleting even more icons

**Review by Trilbyspaceclone — APPROVED** [review]
Other then missing icons that can be fixed at a later date the pr seems good

**Comment by lolman360** [comment]
sprites fixed

**Review by Trilbyspaceclone — APPROVED** [review]

**Pull request merged**